config/repos.yaml

```diff
@@ -20,6 +20,8 @@ theforeman/foreman:
   redmine: foreman
   redmine_required: true
   link_to_redmine: true
+  directory_labels:
+    webpack: ui
```

Comment:
Perhaps also `app/assets`? I know we don't want to add new code there but changes to existing code still occur

Reply:
Right now we can't do that. We are doing [lookup on the first parent folder](https://github.com/theforeman/prprocessor/blob/master/github/pull_request.rb#L184).

Unless we move it from `app/assets` to `foreman_assets` or something similar we have to change the implementation.

Reply:
I tell you what. Maybe I add CODEOWNERS and specify that folder there.
Will that be sufficient?

Reply:
@ripcurld is correct that we limit to a path at the root level. I wrote it that way because it was sufficient for community-templates but I wouldn't mind if we changed the implementation to a more flexible system. Of course we can do that in 2 stages. This change in itself makes sense.

We should also look at the code that applies the labels. Currently it doesn't retrieve the labels before applying them. That leads to a label being assigned that was already assigned. Oddly enough github does log this but since it doesn't email, I haven't looked too much into it.

Reply:
That's what I meant with:

> Of course we can do that in 2 stages. This change in itself makes sense.

Reply:
Yeah I know, I guess I was just :+1:ing that